client/src/components/Ratings/RatingBreakdown/RatingOverview.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import { useMeta } from '@Contexts/ReviewMeta';
+import { useMeta } from '../../../contexts/ReviewMeta';
```

Comment:
I am assuming we are going to need to start doing this with all files moving forward, correct?